docs/_includes/generated/yarn_config_configuration.html

```diff
@@ -77,5 +77,10 @@
             <td style="word-wrap: break-word;">(none)</td>
             <td>A comma-separated list of tags to apply to the Flink YARN application.</td>
         </tr>
+        <tr>
+            <td><h5>yarn.file-replication</h5></td>
+            <td style="word-wrap: break-word;">-1</td>
+            <td>Number of file replication of each local resource file. If it is not configured, Flink will use the default replication value in hadoop configuration.</td>
```

Comment:
state specifically which Hadoop config key is used as default: `yarn.sharedcache.nm.uploader.replication.factor`  but not the normal file replicate key: `dfs.replication`